README.md

```diff
@@ -59,3 +59,9 @@ optional arguments:
                         ERROR)
   -v, --verbose         Show detailed info for each resource checked
 ```
+
+## Namespaces
```

Comment:
Hi @younnoh - thanks for writing this.  I think this should be on the import-export tool (https://github.com/fcrepo4-labs/fcrepo-import-export) instead, not the verify tool